app/src/test/java/com/android/streetworkapp/model/progression/ProgressionModelTest.kt

```diff
@@ -0,0 +1,14 @@
+package com.android.streetworkapp.model.progression
+
+import kotlinx.coroutines.test.runTest
+import org.junit.Test
+
+class ProgressionModelTest {
+
+  @Test
+  fun getOrAddProgressionCallsRepository() = runTest {
```

Comment:
this is pointless, if this is what you needed to get over the 80% coverage, we can just remove the rule and merge your branch instead. If we modify the point system, it will be annoying

Reply:
The intention of the test is unclear, I apologize. The objective is to make sure that the hierarchy of the points is respected. The rationale is that we want user to join an existing event rather than creating one themself, and that adding a friend should be the action with the most points, as this is the main objective of the app. The test as been updated to be more clear, if you want to check.